File: boj_codes/C++17/10871.cpp
// https://www.acmicpc.net/problem/10871
// First Written : 20210118
// Last Modified : 20210118

#include <cstdio>

int main(){
    int N, max;
    int num;

    scanf("%d %d", &N, &max);
    while (N--){
        scanf("%d", &num);
        if(num < max)
            printf("%d ", num);
    }
    


    return 0;
}

File: old_contents/boj_codes/Python3/10871.py
#https://www.acmicpc.net/problem/10871
# 20200909

import sys

N, max = map(int, sys.stdin.readline().split())

seq = list(map(int, sys.stdin.readline().split()))

for num in seq:
    if num < max:
        print(num, end = ' ')
